scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,210 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script helps develop for the contributor dashboard."""
+
+from __future__ import annotations
+
+import hashlib
+import json
+import os
+
+from core import feconf
+from core.constants import constants
+
+import firebase_admin
+from firebase_admin import auth as firebase_auth
+
+import requests
+from typing import Any, Dict, List
+
+FIREBASE_AUTH_EMULATOR_HOST = 'localhost:%s' % feconf.FIREBASE_EMULATOR_PORT
+os.environ['FIREBASE_AUTH_EMULATOR_HOST'] = FIREBASE_AUTH_EMULATOR_HOST
+FIREBASE_SIGN_IN_URL = (
+    'http://' + FIREBASE_AUTH_EMULATOR_HOST +
+    '/identitytoolkit.googleapis.com/v1/accounts:signInWithPassword')
+
+SUPER_ADMIN_EMAIL = feconf.ADMIN_EMAIL_ADDRESS
+SUPER_ADMIN_USERNAME = 'a'
+SUPER_ADMIN_ROLES = [feconf.ROLE_ID_CURRICULUM_ADMIN,
+        feconf.ROLE_ID_TRANSLATION_ADMIN, feconf.ROLE_ID_QUESTION_ADMIN]
+
+CONTRIBUTOR_EMAIL = 'contributor@example.com'
+CONTRIBUTOR_USERNAME = 'b'
+
+CLASSROOM_NAME = 'math'
+CLASSROOM_URL_FRAGMENT = 'math'
+
+
+class ContributorDashboardDebugRequests():
```

Comment:
```suggestion
class ContributorDashboardDebugInitializer():
```
If the main external-facing API is "populate_data()", then something like this seems more appropriate.

Reply:
It makes sense. Thanks